assets/admin/tour/lesson-tour/steps.js

```diff
@@ -63,6 +119,34 @@ export default function getTourSteps() {
 			referenceElements: {
 				desktop: '',
 			},
+			action: async () => {
+				performStepActionsAsync( [
+					// Focus on question block.
+					{
+						action: () => {
+							focusOnQuestionBlock();
+						},
+					},
+					// Click on type selector.
```

Comment:
> the dropdown for selecting question type should open with having true-false focused and selected

In this case, I just opened the dropdown and didn't change the type automatically. I imagine with this the user would already understand this step and we reduce code complexity. Does it make sense? Notice that I also tweaked the text for this: https://github.com/Automattic/sensei/pull/7538/commits/4fb8925b15b20edabdf9399a68cc1d06309fc110